src/App.module.scss

```diff
@@ -47,3 +47,18 @@
   padding: 14px 40px;
   text-decoration: none;
 }
+
+.marsbutton {
+  font-family: 'Orbitron', sans-serif;
```

Comment:
Can we use some of the shared fonts for this?
(or add a new shared font if we need to)

ie something from `generalStyles/scss`

Reply:
it is from general styles line 10 "  font-family: 'Orbitron', sans-serif;"

Reply:
yep - but we want to be using Mixins and constants from that file.
Not just duplicating the code.

The idea is that if we ever wanted to change the fonts / colours etc then we could just change a couple of lines in generalStyles and everything would be done.

Reply:
could you take a look at this one too?